stage/yaml/eoy-dec2-2025.yaml

```
@@ -65,5 +64,46 @@
       - { displayed_notifications: [EOY-Dec2-2025-B] }
       - { displayed_notifications: [EOY-Dec2-2025-C] }
       - { displayed_notifications: [EOY-Nov-2025-C] }
-    percent_chance: 33
-    
\ No newline at end of file
+    percent_chance: 100
+
+  - id: EOY-Dec1-2025-6C
```

```suggestion
- id: EOY-Dec1-2025-6C
```
Doesn't need a space here. (I fixed this after merging last time)